resources/web/wwi/MouseEvents.js

```diff
@@ -172,7 +176,8 @@ export default class MouseEvents {
         WbWorld.instance.viewpoint.position = position.add(target);
         WbWorld.instance.viewpoint.updatePosition();
         this._scene.render();
-      } else if (this._state.mouseDown === 3 || this._state.mouseDown === 4) { // both left and right button or middle button to zoom
+        // both left and right button or middle button to zoom
+      } else if (this._state.mouseDown === 3 || this._state.mouseDown === 4) {
```

Comment:
Also here, isn't better to move the comment inside the corresponding if block?
```suggestion
      } else if (this._state.mouseDown === 3 || this._state.mouseDown === 4) {
        // both left and right button or middle button to zoom
```